inlong-sort/sort-connectors/kafka/src/main/java/org/apache/inlong/sort/kafka/table/UpsertKafkaDynamicTableFactory.java

```diff
@@ -231,7 +232,7 @@ public DynamicTableSource createDynamicTableSource(Context context) {
                 Collections.emptyMap(),
                 0,
                 true,
-                null,
+                inlongMetric,
                 null,
```

Comment:
inlong audit object also is null, you can add it, too.

Reply:
OK, I will fix

Reply:
Fixed, PTAL,thanks! @gong